[Fleet] Fix workaround for missing Fleet server policies for cloud (#980)
elastic/observability-docs: docs/en/ingest-management/release-notes/release-notes-7.14.asciidoc

.. In {kib}, open the main menu, then go to **Management > Dev Tools > Console**.

.. In the Console, send the following requests:
+
[source,console]
----
POST .kibana/_delete_by_query?q=ingest-agent-policies.is_default_fleet_server:true
DELETE .kibana/_doc/ingest-agent-policies:policy-elastic-agent-on-cloud
POST .fleet-enrollment-api-keys/_delete_by_query?q=policy-elastic-agent-on-cloud
----

. Go to **Management > {fleet}** to force reloading of the object.

> . On Cloud you could have to restart APM and Fleet server instances.

Reviewer: ```suggestion
. On {ecloud} you might need to restart the APM and {fleet-server} instances.
```
